config/config.txt

```diff
@@ -57,31 +57,31 @@ MENTOR_LEGACY_SYSTEM
 #MENTORS_MOBNAME_ONLY
 
 ## Comment this out to stop locally connected clients from being given the almost full access !localhost! admin rank
-ENABLE_LOCALHOST_RANK
+#ENABLE_LOCALHOST_RANK
```

Comment:
Why? Commenting this out just makes testing a pain. When will anybody be connecting locally on the production server?

Reply:
Cringe moment.

Reply:
I mean I can understand disabling this locally on the production server just to be paranoid, but making me have to comment this back in by default is lame.

Reply:
Forgot to comment this back in, one sec